spring-cloud-kubernetes-examples/kubernetes-circuitbreaker-ribbon-example/readme.md

```diff
@@ -0,0 +1,218 @@
+## Kubernetes Circuit Breaker & Load Balancer Example
+
+This example demonstrates how to use [Hystrix circuit breaker](https://martinfowler.com/bliki/CircuitBreaker.html) and the [Ribbon Load Balancing](http://microservices.io/patterns/client-side-discovery.html). The circuit breaker which is backed with Ribbon will check regularly if the target service is still alive. If this is not loner the case, then a fall back process will be excuted. In our case, the REST `greeting service` which is calling the `name Service` responsible to generate the response message will reply a "fallback message" to the client if the `name service` is not longer replying.
+As the Ribbon Kubernetes client is configured within this example, it will fetch from the Kubernetes API Server, the list of the endpoints available for the name service and loadbalance the request between the IP addresses available
+
+### Running the example
+
+The project can be deployed on the Kubernetes or OpenShift platform using the [Minishift](https://github.com/minishift/minishift) or [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) tool.
```

Comment:
It sounds like `minikube` or `minishift` are some short of deployment tools, required to install this project.

I'd like us to clarify that the example is meant to run on ALL kubernetes or openshift environments, but for development purposes one can use minikube or minishift to install kubernetes/openshift locally, with no fuss.